groups/transitions.yml

```diff
@@ -0,0 +1,2 @@
+# Features for managing transitions of properties, elements, and documents
+name: Transitions
```

Comment:
I think this should be View Transitions with id `view-transitions` - see https://github.com/web-platform-dx/web-features/pull/1771#discussion_r1784056613 for the rationale.